src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -235,7 +237,16 @@ def run(self) -> List[nodes.Node]:  # noqa: WPS210
                 config_path=config.towncrier_draft_config_path,
             )
         except subprocess.CalledProcessError as proc_exc:
-            raise self.error(proc_exc)
+            cmd = ' '.join(proc_exc.cmd)
+            stdout = proc_exc.stdout or 'None'
+            stderr = proc_exc.stdout or 'None'
+            raise self.error(
+                'Command exited unexpectedly\n\n'
+                f'Command: {cmd}\n'
+                f'Exit code: {proc_exc.returncode}\n\n'
```

Comment:
The POSIX term is return code, let's not rename it to keep clear what it is.
```suggestion
                f'Return code: {proc_exc.returncode}\n\n'
```